packages/chain-adapters/src/cosmossdk/thorchain/ThorchainChainAdapter.ts

```diff
@@ -210,7 +212,7 @@ export class ChainAdapter extends CosmosSdkBaseAdapter<KnownChainIds.ThorchainMa
       const msg: ThorchainMsgSend = {
         type: ThorchainMessageType.MsgSend,
         value: {
-          amount: [{ amount, denom: this.denom }],
+          amount: [{ amount, denom: coin === 'THOR.TCY' ? 'tcy' : this.denom }],
```

Comment:
Reusing the `coin` property from chainSpecific since that's what we're already using for `MsgDeposit`, but the denom is `tcy`, not `THOR.TCY` (that's the coin for deposits)

Reply:
Theoretically, we could just instantiate the adapter with the right `denom` but meh, that would include much larger changes around the codebase since `assertGetCosmosSdkChainAdapter` wouldn't work anymore, and nor would `chainAdapterManager.get(chainId)`: we would not be getting an adapter from `ChainId` only, but would have to instantiate a new one by `AssetId` instead.